numpy/core/src/umath/reduction.c

```diff
@@ -524,8 +522,17 @@ PyUFunc_ReduceWrapper(PyArrayObject *operand, PyArrayObject *out,
                   NPY_ITER_NO_SUBTYPE;
     op_flags[1] = NPY_ITER_READONLY |
                   NPY_ITER_ALIGNED;
+    if (wheremask != NULL) {
+        op[2] = wheremask;
+        op_dtypes[2] = PyArray_DescrFromType(NPY_BOOL);
+        if (op_dtypes[2] == NULL) {
+            goto fail;
+        }
+        op_flags[2] = NPY_ITER_READONLY |
+                      NPY_ITER_ALIGNED;
```

Comment:
What's the aligned for here?

Reply:
Yes, I guess for bools that is pretty senseless. Removed.